tools/webtransport/requirements.txt

```diff
@@ -1 +1,2 @@
 aioquic==1.2.0
+cryptography
```

Comment:
This should probably pin its version, though I guess we're effectively already getting it as a dependency of aioquic anyway.

Reply:
Sure, when I wrote this, I was unaware it was a dependency. Anyway, if aioquic is replaced by something else, certificates for webtransport will still be needed (e.g. for wt over http/2 sometime)